sdk/pixKey/pixKey.js

```diff
@@ -0,0 +1,226 @@
+        this.status = status;
+        this.bankCode = bankCode;
+        this.bankName = bankName;
+        this.type = type;
+        this.created = check.datetime(created);
+    }
+}
+
+exports.PixKey = PixKey;
+let resource = {'class': exports.PixKey, 'name': 'PixKey'};
+
+exports.create = async function (report, {user} = {}) {
+    /**
+     *
+     * Create a PixKey object
+     *
+     * @description Create a PixKey linked to a specific account in the Stark Infra API
+     *
+     * Parameters (required):
+     * @param key [PixKey object]: PixKey object to be created in the API.
+     *
+     * Parameters (optional):
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+     *
+     * Return:
+     * @returns PixKey object with updated attributes.
+     *
+     */
+    return rest.postSingle(resource, report, user);
+};
+
+exports.get = async function (id, payerId, { endToEndId, user } = {}) {
+    /**
+     *
+     * Retrieve a PixKey object
+     *
+     * @description Retrieve the PixKey object linked to your Workspace in the Stark Infra API by its id.
+     *
+     * Parameters (required):
+     * @param id [string]: PixKey object unique id. ex: '5656565656565656'
+     * @param payerId [string]: tax id (CPF/CNPJ) of the individual or business requesting the PixKey information. This id is used by the Central Bank to limit request rates. ex: "20.018.183/0001-80".
+     *
+     * Parameters (optional):
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+     *
+     * Return:
+     * @returns PixKey object that corresponds to the given id.
+     *
+     */
+    return rest.getId(resource, id, user, {'payerId': payerId, 'endToEndId': endToEndId});
+};
+
+exports.query = async function ({ limit, after, before, status, tags, ids, type, user } = {}) {
+    /**
+     *
+     * Retrieve PixKeys
+     *
+     * @description Receive a generator of PixKeys objects previously created in the Stark Infra API
+     *
+     * Parameters (optional):
+     * @param limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35
+     * @param after [string, default null]: date filter for objects created after a specified date. ex: '2020-03-10'
+     * @param before [string, default null]: date filter for objects created before a specified date. ex: '2020-03-10'
+     * @param status [list of strings, default null]: filter for status of retrieved objects. Options: "created", "failed", "delivered", "closed", "canceled".
+     * @param ids [list of strings, default null]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+     * @param type [list of strings, default null]: filter for the type of retrieved PixKeys. Options: "fraud", "reversal", "reversalChargeback"
```

Comment:
type não é uma lista de strings. É um string dentre as opções: "cpf", "cnpj", "phone", "email" and "evp"